sound/soc/sof/ipc4.c

```diff
@@ -99,6 +99,10 @@ static int sof_ipc4_check_reply_status(struct snd_sof_dev *sdev, u32 status)
 
 to_errno:
 	switch (status) {
+	case 2:
+	case 15:
+		ret = -EOPNOTSUPP;
+		break;
```

Comment:
Looks like we don't really have a consistent  way of reporting "not supported"

````
ipc3-dtrace.c:          return -EOPNOTSUPP;
ipc4-loader.c:          return -EOPNOTSUPP;
ops.h:  return -ENOTSUPP;
ops.h:  return -ENOTSUPP;
sof-client.c:   return -EOPNOTSUPP;

````

Reply:
I'll update the ops.h, `EOPNOTSUPP` is the preferred one.
```
$ rg EOPNOTSUPP scripts 
scripts/checkpatch.pl
4890:# Folks usually mean EOPNOTSUPP (also called ENOTSUP), when they type ENOTSUPP.
4894:                            "ENOTSUPP is not a SUSV4 error code, prefer EOPNOTSUPP\n" . $herecurr) &&
4896:                           $fixed[$fixlinenr] =~ s/\bENOTSUPP\b/EOPNOTSUPP/;
```

Reply:
@plbossart, https://github.com/thesofproject/linux/pull/4525 for the `ops.h`